script/shard_spec_repo

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env ruby
+
+require 'bundler/inline'
+require 'tmpdir'
+require 'fileutils'
+
+gemfile do
+  gem 'cocoapods', '>= 1.1.0.rc.1'
+  gem 'cocoapods-repo-shard'
+
+  gem 'sequel'
+end
+
+Pod::Config.instance.verbose = true
+
+master = Pod::SourcesManager.master.first
+prefix_lengths = [1, 1, 1]
+
+Pod::Executable.execute_command(:heroku, %w(config:set TRUNK_APP_PUSH_ALLOWED=false), true)
+
+Pod::Command.run(%w(repo update master))
+
+Dir.chdir(master.repo) do
+  pre_shard_sha = Pod::Executable.execute_command(:git, %w(rev-parse HEAD), true).strip
+  Dir.mktmpdir do |old_specs|
+    Dir.chdir(old_specs) do
+      FileUtils.cp_r %w(Specs .gitignore README.md CocoaPods-version.yml), '.'
+      Pod::Executable.execute_command(:git, %w(init .), true)
+      Pod::Executable.execute_command(:git, %w(remote add old-specs https://github.com/CocoaPods/Old-Specs.git), true)
+      Pod::Executable.execute_command(:git, %w(add -A), true)
+      Pod::Executable.execute_command(:git, ['commit',
+                                             '-m', "Create a copy of CocoaPods/Specs@#{pre_shard_sha}",
+                                             '-m', 'Please see https://blog.cocoapods.org/Sharding/ for more information',
+                                             '-m', "Please use https://github.com/CocoaPods/Specs if you're running CocoaPods 1.0+"],
+                                      true)
+      Pod::Executable.execute_command(:git, %w(push old-specs master), true)
+    end
+  end
+end
+
+Pod::Command.run(%W(repo shard master --lengths=#{prefix_lengths.join(',')}))
+
+# creating from scratch to avoid re-serializing any stale data
+Pod::Source::Metadata.from_file(master.metadata_path).tap do |metadata|
+  metadata.minimum_cocoapods_version = Pod::Version.new('1.0.0')
```

Comment:
Is this minimum required version correct? As in, can 1.0.0 work with sharded repos?